code/modules/vehicles/ridden.dm

```diff
@@ -36,27 +36,26 @@
 	return ..()
 
 /obj/vehicle/ridden/attackby(obj/item/I, mob/user, params)
-	if(key_type && !is_key(inserted_key) && is_key(I))
-		if(user.transferItemToLoc(I, src))
-			to_chat(user, "<span class='notice'>You insert \the [I] into \the [src].</span>")
-			if(inserted_key)	//just in case there's an invalid key
-				inserted_key.forceMove(drop_location())
-			inserted_key = I
-		else
-			to_chat(user, "<span class='warning'>[I] seems to be stuck to your hand!</span>")
-		return
-	return ..()
+	if(!(key_type && !is_key(inserted_key) && is_key(I)))
+		return ..()
+	if(user.transferItemToLoc(I, src))
+		to_chat(user, "<span class='notice'>You insert \the [I] into \the [src].</span>")
+		if(inserted_key)	//just in case there's an invalid key
+			inserted_key.forceMove(drop_location())
+		inserted_key = I
+	else
+		to_chat(user, "<span class='warning'>[I] seems to be stuck to your hand!</span>")
```

Comment:
```suggestion
	if(!user.transferItemToLoc(I, src))
		to_chat(user, "<span class='warning'>[I] seems to be stuck to your hand!</span>")
		return
	to_chat(user, "<span class='notice'>You insert \the [I] into \the [src].</span>")
	if(inserted_key)	//just in case there's an invalid key
		inserted_key.forceMove(drop_location())
	inserted_key = I
```